protocol/src/main/protobuf/Coordination.proto

```diff
@@ -18,6 +18,7 @@ message DrillbitEndpoint{
       OFFLINE = 3;
   }
   optional State state = 7;
+  optional int32 http_port = 8;
```

Comment:
Please compile protobuf for c++ client as well (Instructions are [here](https://github.com/apache/drill/blob/master/contrib/native/client/readme.linux)). It would be really helpful if you can put a note in [readme.txt of this package](https://github.com/apache/drill/blob/master/protocol/readme.txt) for future dev to compile c++ client protobufs whenever a change is made in any of .proto files.